app/routers/analyze.py

```
@@ -0,0 +1,67 @@
+import shutil
+from fastapi import APIRouter, UploadFile, File, HTTPException
+from app.utils.parsers import ParserFactory
+from app.utils.ai_client import AIClient
+from app.utils.risk_parser import RiskParser
+
+router = APIRouter()
+
+UPLOAD_DIR = "uploads"
+
+@router.post("/extract_text/")
+async def extract_text_from_document(file: UploadFile = File(...)):
```

Ruff (0.8.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)